ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/PresentmentDetails.kt

```diff
@@ -0,0 +1,13 @@
+package com.braintreepayments.api.shopperinsights
+
+
+/**
+ * Initializes a new PresentmentDetails instance
+ *
+ * @param treatmentName The experiment or treatment name
+ * @param type An ExperimentType that is either a control or test type
+ */
+data class PresentmentDetails (
+    val treatmentName: String,
```

Comment:
The parameters here don't seem to match the ticket description - can you confirm? It should align with iOS too.